packages/nodes-base/nodes/ItemLists/V3/helpers/utils.ts

```diff
@@ -57,3 +64,26 @@ export const prepareFieldsArray = (fields: string | string[], fieldName = 'Field
 		`The \'${fieldName}\' parameter must be a string of fields separated by commas or an array of strings.`,
 	);
 };
+
+const returnRegExp = /\breturn\b/g;
```

Comment:
should this var be inside function?

Reply:
should be fine to leave it here. just an old habit of mine to cache constants outside functions. doesn't make much of a difference now a days.